exchanges/orderbook/orderbook.go

```diff
@@ -202,6 +201,13 @@ func (s *Service) Retrieve(exchange string, p currency.Pair, a asset.Item) (*Bas
 	return book.Retrieve()
 }
 
+// GetDepth returns the concrete book of which Base is a copy
+// This is probably useful for immutably monitoring orderbook health and state
+// whereas FetchOrderbook would trigger a refresh.
```

Comment:
```suggestion
// GetDepth returns the concrete book allowing the caller to stream orderbook changes
```
FetchOrderbook desn't necessarily trigger a refresh

Reply:
Okay. So s/ would / might / and add context.

Reply:
Done, exactly as per suggestion.